server/oam/kubernetes/workloads/APIService_definitions.k8s.json

```diff
@@ -0,0 +1 @@
+{"kind":"WorkloadDefinition","apiVersion":"core.oam.dev/v1alpha1","metadata":{"name":"APIService.K8s","creationTimestamp":null},"spec":{"definitionRef":{"name":"apiservice.k8s.meshery.layer5.io"},"metadata":{"@type":"pattern.meshery.io/k8s","isCustomResource":"false","k8sAPIVersion":"apiregistration.k8s.io/v1","k8sKind":"APIService","namespaced":"false","version":"v1.25.0"}}}
```

Comment:
We seem to be missing a directory with the system version number, like `v1.25.0`.